### PR TITLE
HBASE-25659 [hbck2] Add hbase-operator-tools command for scheduleSCPsForUnknownServers

### PR DESCRIPTION
**Pull request by petersomogyi**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  9s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 34s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 27s |  master passed  |
| +1 :green_heart: |  compile  |   0m 19s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 11s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 12s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 20s |  the patch passed  |
| -1 :x: |  checkstyle  |   0m  4s |  hbase-hbck2: The patch generated 4 new + 0 unchanged - 0 fixed = 4 total (was 0)  |
| -1 :x: |  checkstyle  |   0m  4s |  root: The patch generated 4 new + 0 unchanged - 0 fixed = 4 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 11s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 42s |  hbase-hbck2 in the patch passed.  |
| -1 :x: |  unit  |  30m 28s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |  39m 46s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hbase.TestRegionsMerger |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/84 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile xml |
| uname | Linux 0e3113ef233e 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / b4f4676 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/artifact/yetus-precommit-check/output/diff-checkstyle-hbase-hbck2.txt |
| checkstyle | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/artifact/yetus-precommit-check/output/diff-checkstyle-root.txt |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/artifact/yetus-precommit-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/testReport/ |
| Max. process+thread count | 968 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/1/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 44s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 28s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 29s |  master passed  |
| +1 :green_heart: |  compile  |   0m 20s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 10s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m  8s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 10s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 53s |  hbase-hbck2 in the patch passed.  |
| -1 :x: |  unit  |  28m  2s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |  37m  0s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hbase.TestRegionsMerger |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/2/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/84 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile xml |
| uname | Linux 99fd29877839 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / b4f4676 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/2/artifact/yetus-precommit-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/2/testReport/ |
| Max. process+thread count | 955 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/2/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  8s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 34s |  master passed  |
| +1 :green_heart: |  compile  |   0m 22s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 13s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 13s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   0m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 22s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 22s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 10s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 11s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 54s |  hbase-hbck2 in the patch passed.  |
| -1 :x: |  unit  |  33m 24s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |  43m 17s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hbase.TestRegionsMerger |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/3/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/84 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile xml |
| uname | Linux b68370015014 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / b4f4676 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/3/artifact/yetus-precommit-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/3/testReport/ |
| Max. process+thread count | 962 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/3/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 1]
```diff
@@ -659,6 +668,16 @@ private static void usageScheduleRecoveries(PrintWriter writer) {
     writer.println("   Command support added in hbase versions 2.0.3, 2.1.2, 2.2.0 or newer.");
   }
 
+  private static void usageRecoverUnknown(PrintWriter writer) {
+    writer.println(" " + RECOVER_UNKNOWN);
+    writer.println("   Schedule ServerCrashProcedure(SCP) RegionServers that are reported");
```
Comment (wchevreuil) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:673]: nit: "Schedule ServerCrashProcedure(SCP) for RegionServers..."

**Review by wchevreuil — APPROVED** [review]

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 44s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  spotbugs  |   0m  0s |  spotbugs executables are not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 10s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   0m 27s |  master passed  |
| +1 :green_heart: |  compile  |   0m 19s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 12s |  master passed  |
| +1 :green_heart: |  javadoc  |   0m 12s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 27s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m  8s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  javadoc  |   0m 11s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 40s |  hbase-hbck2 in the patch passed.  |
| +1 :green_heart: |  unit  |   6m 43s |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m  9s |  The patch does not generate ASF License warnings.  |
|  |   |  16m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/4/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-operator-tools/pull/84 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs checkstyle compile xml |
| uname | Linux 0ff0407c74cd 5.4.0-1025-aws #25~18.04.1-Ubuntu SMP Fri Sep 11 12:03:04 UTC 2020 x86_64 GNU/Linux |
| Build tool | maven |
| git revision | master / b4f4676 |
| Default Java | Oracle Corporation-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/4/testReport/ |
| Max. process+thread count | 1217 (vs. ulimit of 5000) |
| modules | C: hbase-hbck2 . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-Operator-Tools-PreCommit/job/PR-84/4/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**